posthog/api/organization.py

```diff
@@ -132,12 +132,12 @@ def get_metadata(self, instance: Organization) -> Dict[str, Union[str, int, obje
         except ImportError:
             return output
 
-        output["taxonomy_set_events_count"] = EnterpriseEventDefinition.objects.exclude(
-            description="", tagged_items__isnull=True
-        ).count()
-        output["taxonomy_set_properties_count"] = EnterprisePropertyDefinition.objects.exclude(
-            description="", tagged_items__isnull=True
-        ).count()
+        # output["taxonomy_set_events_count"] = EnterpriseEventDefinition.objects.exclude(
+        #     description="", tagged_items__isnull=True
+        # ).count()
+        # output["taxonomy_set_properties_count"] = EnterprisePropertyDefinition.objects.exclude(
+        #     description="", tagged_items__isnull=True
+        # ).count()
```

Comment:
Let's just remove this instead of commenting-out, this definitely isn't something we want to select _every time_ an organization is serialized

Reply:
will do that as a follow up after incident

Reply:
This is now causing commits to fail with 

```
❯ Running tasks...
  ✔ Running tasks for *.{json,yaml,yml,css,scss}
  ✔ Running tasks for (!(plugin-server)/**).{js,jsx,mjs,ts,tsx}
  ✔ Running tasks for (plugin-server/**).{js,jsx,mjs,ts,tsx}
  ❯ Running tasks for !(posthog/hogql/grammar/*)*.{py,pyi}
    ✔ black
    ✖ ruff [FAILED]
posthog/api/organization.py:131:35: F401 [*] `ee.models.EnterpriseEventDefinition` imported but unused
posthog/api/organization.py:131:62: F401 [*] `ee.models.EnterprisePropertyDefinition` imported but unused
Found 2 errors.
```